docs/resources/vrf.md

```diff
@@ -7,81 +10,438 @@ description: |-
+  owner_tag                             = "owner_tag_1"
+  policy_control_enforcement_direction  = "egress"
+  policy_control_enforcement_preference = "enforced"
+  relation_to_bgp_timers = {
+    annotation      = "annotation_1"
+    bgp_timers_name = aci_bgp_timers.example.name
+    annotations = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+    tags = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+  }
+  relation_to_monitoring_policy = {
+    annotation             = "annotation_1"
+    monitoring_policy_name = aci_monitoring_policy.example.name
+    annotations = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+    tags = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+  }
+  relation_to_bgp_address_family_contexts = [
+    {
+      address_family                  = "ipv4-ucast"
+      annotation                      = "annotation_1"
+      bgp_address_family_context_name = aci_bgp_address_family_context.example.name
+      annotations = [
+        {
+          key   = "key_0"
+          value = "value_1"
+        }
+      ]
+      tags = [
+        {
+          key   = "key_0"
+          value = "value_1"
+        }
+      ]
+    }
+  ]
+  relation_to_eigrp_address_family_contexts = [
+    {
+      address_family                    = "ipv4-ucast"
+      annotation                        = "annotation_1"
+      eigrp_address_family_context_name = aci_eigrp_address_family_context.example.name
+      annotations = [
+        {
+          key   = "key_0"
+          value = "value_1"
+        }
+      ]
+      tags = [
+        {
+          key   = "key_0"
+          value = "value_1"
+        }
+      ]
+    }
+  ]
+  relation_to_end_point_retention_policy = {
+    annotation                      = "annotation_1"
+    end_point_retention_policy_name = aci_end_point_retention_policy.example.name
+    annotations = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+    tags = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+  }
+  relation_to_l3out_route_tag_policy = {
+    annotation                  = "annotation_1"
+    l3out_route_tag_policy_name = aci_l3out_route_tag_policy.example.name
+    annotations = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
+    tags = [
+      {
+        key   = "key_0"
+        value = "value_1"
+      }
+    ]
   }
+  relation_to_address_family_ospf_timers = [
+    {
+      address_family   = "ipv4-ucast"
+      annotation       = "annotation_1"
+      ospf_timers_name = aci_ospf_timers.example.name
+      annotations = [
+        {
+          key   = "key_0"
+          value = "value_1"
+        }
+      ]
+      tags = [
+        {
+          key   = "key_0"
+          value = "value_1"
+        }
+      ]
+    }
+  ]
+  relation_to_wan_vpn = {
```

Comment:
```suggestion
  relation_to_sdwan_vpn = {
```

Reply:
named it wan vpn because of the naming in UI, diverge from UI in this case?